public/js/html.js

```diff
@@ -0,0 +1,81 @@
+$(function() {
+	$("#login-form").show();
+	$("#register-form").hide();
+    $('#login-form-link').click(function(event) {
+		$("#register-form").hide();
+		$("#login-form").show();
+		$('#register-form-link').removeClass('active');
+		$(this).addClass('active');
+		event.preventDefault();
+	});
+	$('#register-form-link').click(function(event) {
+		$("#login-form").hide();
+		$("#register-form").show();
+		$('#login-form-link').removeClass('active');
+		$(this).addClass('active');
+		event.preventDefault();
+	});
+
+	const modal = document.getElementById('myModal');
+	const login = document.getElementById("login-start");
+	const span = document.getElementsByClassName("close")[0];
+	login.onclick = function() {
+		modal.style.display = "block";
+	}
+	span.onclick = function() {
+		modal.style.display = "none";
+	}
+	window.onclick = function(event) {
+		if (event.target == modal) {
+			modal.style.display = "none";
+		}
+	}
+
+	$("#login-submit").on("click",function(){
+		const username = $("#username").val().trim();
+		const password = $("#password").val().trim();
+		
+		function getLogin(username){
```

Comment:
Defining getLogin as a function in this code doesn't serve any purpose, you should just put the code thats in the function directly into the handler.